documentation/release_notes.rst

```diff
@@ -8,6 +8,69 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.5.0
+===============
+
+New Features
+------------
+- Added new ``histogram`` algorithms for generating a histogram from an input sequence into
+  an output sequence representing either equally spaced or user-defined bins.
+  These algorithms are currently only available for device execution policies.
+- Added ability to avoid group reduction operations for 64-bit arithmetic data types.
```

Comment:
This is not really a feature but a workaround to the known issue described at lines 64-66. Maybe the best way to refer to it is this wording we coined earlier (supposed to be right after the problem description) :
```
  For a workaround, define the ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION macro to 1
  before including oneDPL header files.
```

Reply:
We might as well add something to the bug fix section, e.g. "Added a workaround for reduction algorithm failures with 64-bit data types; see the Known Issues below" or alike.

Reply:
I added the workaround's description to the list of known issues. I also suggest to highlight it in the bug fix section, when it's addition is mentioned, to simplify the search.